src/command_modules/azure-cli-acr/README.rst

```diff
@@ -84,23 +84,41 @@ Update a container registry
 
     Command
         az acr update: Update a container registry.
-
+    
     Arguments
         --name -n      [Required]: Name of container registry.
-        --disable-admin          : Disable admin user.
-        --enable-admin           : Enable admin user.
+        --admin-user-enabled -a  : Whether the admin user account is enabled.  Allowed values: false,
```

Comment:
If `--admin-user-enabled` becomes a flag, is it possible for the user to disable the admin user? I tried using `store_true` and it seems to be impossible to disable the admin user without explicitly using `--set adminUserEnabled=false`.